### DISCUSSION TITLE
Unintentional scrolling when trying to zoom in on trackpad

### DISCUSSION THREAD

**Discussion by FroznAnna** [flxzt/rnote#1300] [Development]
I love rnote and use it for all my lectures.
One of my modules has a 120+ page pdf which i have inported into rnote so I can write over it.
Only issue is when i try to zoom in on the trackpad using the pinch action, it sends me to narnia. It zooms about 50 pages in one second.
Do you know how to turn this off?

**Comment by Doublonmousse** [comment]
I wonder if that's an instance of issue #885

**Reply by FroznAnna** [reply]
Yeah youre right sorry, there doesnt seem to be any solution for this atm. Maybe they can implement location lock or smt idk
